galaxy_ng/tests/integration/api/test_container_delete.py

```diff
@@ -0,0 +1,155 @@
+    repo_response = client(f"{api_prefix}{repo_href}")
+
+    # Grab <latest_version_href> field from this Container Push Repo Instance
+    latest_version = repo_response["latest_version_href"]
+
+    # Filter List Content List by the latest version found above
+    content_list = client(
+        f"{api_prefix}/pulp/api/v3/content/?repository_version={latest_version}",
+    )
+
+    # Check content before deleting
+    assert len(content_list["results"]) > 0
+
+    # Delete repository, contents, and artifacts
+    delete_response = client(f"{api_prefix}/v3/"
+                             "plugin/execution-environments/repositories/alpine/", method='DELETE')
+    resp = wait_for_task(client, delete_response, timeout=10000)
+    assert resp["state"] == "completed"
+
+    # Ensure content list is empty by checking each content href
+    content_hrefs = [item["pulp_href"] for item in content_list["results"]]
+
+    for item in content_hrefs:
+        failed = None
+        try:
+            client(f"{api_prefix}{item}")
+            failed = False
+        except GalaxyError as ge:
+            if ge.http_code in [403, 404]:
+                failed = True
+            else:
+                raise Exception(ge)
+
+        assert failed
+
+
+@pytest.mark.standalone_only
+def test_shared_content_is_not_deleted(ansible_config):
+    # Pull alpine image
+    subprocess.check_call(["docker", "pull", "alpine"])
+
+    # Tag the image
+    subprocess.check_call(["docker", "tag", "alpine", "localhost:5001/alpine1:latest"])
+
+    # Login to local registy with tls verify disabled
+    subprocess.check_call(["docker", "login", "-u", "admin", "-p", "admin", "localhost:5001"])
+
+    # Push image to local registry
+    subprocess.check_call(["docker", "push", "localhost:5001/alpine1:latest"])
+
+    # Copy 'alpine1' and rename to 'alpine2'
+    subprocess.check_call(["docker", "tag", "alpine", "localhost:5001/alpine2:latest"])
+    subprocess.check_call(["docker", "push", "localhost:5001/alpine2:latest"])
+
+    # Get an API client running with admin user credentials
+    client = get_client(
+        config=ansible_config("admin"),
+        request_token=True,
+    )
+    api_prefix = client.config.get("api_prefix").rstrip("/")
+
+    # Select the distribution for alpine1 and alpine2.
+    distro_response1 = client(
+        f"{api_prefix}/pulp/api/v3/distributions/"
+        "container/container/?base_path=alpine1",
+    )
+    distro_response2 = client(
+        f"{api_prefix}/pulp/api/v3/distributions/"
+        "container/container/?base_path=alpine2",
+    )
+    assert distro_response1["results"][0]["base_path"] == 'alpine1'
+    assert distro_response2["results"][0]["base_path"] == 'alpine2'
+
+    # Grab the repository href from the json and make get request
+    repo_href_1 = (distro_response1["results"][0]["repository"]).replace("/api/automation-hub", "")
```

Comment:
You shouldn't have to replace `/api/automation-hub`. The href it gives you will work by itself